tike/trajectory.py

```diff
@@ -107,7 +107,7 @@ def euclidian_dist_approx(theta, v, h, r=0.75):
     return np.abs(t1) * r + np.sqrt(v1**2 + h1**2)
 
 
-def discrete_trajectory(trajectory, tmin, tmax, xstep, tstep, tkwargs={}):
+def discrete_trajectory(trajectory, tmin, tmax, xstep, tstep, tkwargs=None):
```

Comment:
tkwargs ?= **kwargs

Reply:
They are not the same. `tkwargs` are the keyword args to be passed to the `trajectory`. They are not the keyword args to be used for `discrete_helper` or `discrete_trajectory`.